docs/consume-events.md

```diff
@@ -27,6 +27,10 @@ This [example documentation (must be part of VA GitHub organization to view)](ht
 
 ![A diagram showing various scenarios indicating whether a system would need an ESECC. If you are already on the Lighthouse Delivery Infrastructure (LHDI), no ESECC is required. If you are on AWS GovCloud but outside of LHDI, check to see if an ESECC is required. If you are on Non-AWS Cloud (e.g. Azure), check to see if an ESECC is required. If you are on-premises, e.g., VistA, an ESECC is required.](img/Client-Environments-ESECC-Decision-Circles.svg)
```

Comment:
Same comment as on the other doc re if this diagram has been verified recently.

Reply:
Addressed this in a previous comment. Let me know if you agree.

Reply:
Yes, as noted above: I think a new ticket to review those sections, and update if needed, would be a good idea. Our guidance from LHDI on ESECCs has been shifting around over the past few months and this information may still be correct, or it may not.

Reply:
Added [#3044](https://app.zenhub.com/workspaces/event-bus-team-632b50a9a7bf8400277f2cf8/issues/gh/department-of-veterans-affairs/ves/3044) to fix the ESECC section.